contracts/wormhole/contracts/mapping-token/v2/MappingTokenFactory.sol

```diff
@@ -241,133 +197,22 @@ contract MappingTokenFactory is Initializable, Ownable, DailyLimit, ICrossChainF
-        require(salt2MappingToken[salt] == address(0), "MappingTokenFactory:contract has been deployed");
-        mappingToken = deploy(salt, tokenType);
-        IMappingToken(mappingToken).initialize(
-            string(abi.encodePacked(name, "[", bridgedChainName, ">")),
-            string(abi.encodePacked("x", symbol)),
-            decimals);
-
-        // save the mapping tokens in an array so it can be listed
-        allMappingTokens.push(mappingToken);
-        // map the originToken to mappingInfo
-        salt2MappingToken[salt] = mappingToken;
-        // map the mappingToken to origin info
-        mappingToken2OriginalInfo[mappingToken] = OriginalInfo(bridgedChainPosition, tokenType, backingAddress, originalToken);
-        emit IssuingERC20Created(backingAddress, originalToken, mappingToken);
-    }
-
-    /**
-     * @notice issue mapping token, only can be called by inboundLane
-     * @param backingAddress the backingAddress which send this message
-     * @param originalToken the original token address
-     * @param recipient the recipient of the issued mapping token
-     * @param amount the amount of the issued mapping token
-     */
-    function issueMappingToken(
-        address backingAddress,
-        address originalToken,
-        address recipient,
-        uint256 amount
-    ) public onlyInBoundLane(backingAddress) whenNotPaused {
-        uint32 bridgedChainPosition = IMessageCommitment(msg.sender).bridgedChainPosition();
-        address mappingToken = getMappingToken(bridgedChainPosition, backingAddress, originalToken);
-        require(mappingToken != address(0), "MappingTokenFactory:mapping token has not created");
-        require(amount > 0, "MappingTokenFactory:can not receive amount zero");
-        expendDailyLimit(mappingToken, amount);
-        if (guard != address(0)) {
-            IERC20(mappingToken).mint(address(this), amount);
-            require(IERC20(mappingToken).approve(guard, amount), "MappingTokenFactory:approve token transfer to guard failed");
-            IInboundLane.InboundLaneNonce memory inboundLaneNonce = IInboundLane(msg.sender).inboundLaneNonce();
-            // todo we should transform this messageId to bridged outboundLane messageId
-            uint256 messageId = IInboundLane(msg.sender).encodeMessageKey(inboundLaneNonce.last_delivered_nonce);
-            IGuard(guard).deposit(messageId, mappingToken, recipient, amount);
-        } else {
-            IERC20(mappingToken).mint(recipient, amount);
-        }
-    }
-
-    /**
-     * @notice burn mapping token and unlock remote original token, waiting for the confirm
-     * @param bridgedLanePosition the bridged lane position to send the unlock message
-     * @param mappingToken the burt mapping token address
-     * @param recipient the recipient of the remote unlocked token
-     * @param amount the amount of the burn and unlock
-     */
-    function burnAndRemoteUnlockWaitingConfirm(
+    function _sendMessage(
```

Comment:
This looks like to be the same with the `_sendMessage ` in Backing contract, we may have a comparison with their implementation.